portal_spa/src/components/layout/AttributeIndexLayout.tsx

```diff
@@ -0,0 +1,209 @@
+      cred.name_en?.toLowerCase().includes(q) ||
+      cred.org_name?.toLowerCase().includes(q) ||
+      cred.ap_slug?.toLowerCase().includes(q)
+    );
+  });
+
+  const isMainPage = useMatch("/attribute-index");
+  const location = useLocation();
+
+  useEffect(() => {
+    // Clearing search searchQuery when navigating to a different page (mainly from main page to details page so the search bar is cleared)
+    setsearchQuery("");
+  }, [location]);
+
+  function SearchDropdown({
+    results,
+    onSelect,
+  }: {
+    results: Credential[];
+    onSelect: () => void;
+  }) {
+    if (results.length === 0) return null;
+
+    return (
+      <div className="absolute z-50 bg-white border rounded shadow w-full mt-1 max-h-80 overflow-auto">
+        <ul className="divide-y divide-gray-100">
+          {results.map((cred) => {
+            const { credential_id, ap_slug, environment, name_en, id } = cred;
+            if (!credential_id || !ap_slug || !environment) return null;
+
+            return (
+              <li key={id} className="p-2 hover:bg-gray-100">
+                <Link
+                  to={`/attribute-index/credentials/${environment}/${ap_slug}/${credential_id}`}
+                  onClick={onSelect}
+                  className="block text-sm font-medium text-gray-900"
+                >
+                  {name_en}
+                </Link>
+                <p className="text-xs text-gray-500">
+                  {credential_id} ({environment})
+                </p>
+              </li>
+            );
+          })}
+        </ul>
+      </div>
+    );
+  }
+
+  function capitalizeFirstLetter(string: string): string {
+    return string.charAt(0).toUpperCase() + string.slice(1);
+  }
+
+  return (
+    <div className="flex">
+      {/* Sidebar */}
+
+      <aside className="w-72 border-r p-4">
+        <Link to="/attribute-index" className="block mb-4">
+          <h1 className="text-xl font-bold mb-4">Attribute Index</h1>
+        </Link>
+        <Separator className="my-4" />
+        <ScrollArea className="h-screen pr-2">
+          <Accordion
+            type="multiple"
+            defaultValue={["production", "staging", "demo"]}
+          >
+            {Object.entries(grouped).map(([env, aps]) => (
+              <AccordionItem value={env} key={env}>
+                <AccordionTrigger>
+                  <Link to={`/attribute-index/environments/${env}`}>
+                    {capitalizeFirstLetter(env)} Environment
+                  </Link>
+                </AccordionTrigger>
+                <AccordionContent>
+                  <Accordion type="multiple">
+                    {Object.entries(aps).map(([key, creds]) => {
+                      const referenceCredential = creds[0]; // Use the first credential as reference for org_slug and org_name
+                      if (!referenceCredential) return null;
+                      const organizationName = `${referenceCredential.org_name}`;
```

Comment:
Template string is redundant on this line.

```suggestion
                      const organizationName = referenceCredential.org_name;
```